ufobalapp/views_auth.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/python
+# -*- coding: UTF-8 -*-
 import json
```

Comment:
Není tohle už trochu zastaralý? V pythonu 3 pro to neznám důvod, ale klidně mě pouč

Reply:
mam pocit ze jsem taky mel nejakej problem s diakritikou, ale jeste to zkusim

Reply:
tak to je divný, když tak napiš co, možná můžeme něco aktualizovt, ale produkčně to rozhodně běží na 3

Reply:
uz jsem to zvladl rozjet, stahl jsem si predbuildeny wheely